### PR TITLE
Add Naseem Raad

### PR DESCRIPTION
**Pull request by nzraad**

### CODE REVIEW

File: SIGNEES.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-* Micah Cambre (@asuh)
-* Shavonn Brown (@shavonn)
-* Lyn Phan (@lynphan)
-* Jeremy Bardrof (@jeremy-bardrof)
-* Hepp Maccoy (@hepp)
-* Mathieu Couette (@MathieuCouette)
-* Scott Hawkins (@echoboomer)
-* Leo Li (@leo-at-biteable)
-* Patrick Rosales (@patrickrosales)
+- Tommy Fotak (@tojofo)
```
Comment (biteablez) [inline SIGNEES.md:3]: @nzraad What do you have against asterisks?

Comment (nzraad) [reply SIGNEES.md:3]: 😆 I totally missed that. I guess it's my VSCode Markdown linter that was totally against it. Let me remove these plugins before they become chaotic to the Biteable GitHub Repos! [https://github.com/DavidAnson/vscode-markdownlint](https://github.com/DavidAnson/vscode-markdownlint)

Comment (biteablez) [reply SIGNEES.md:3]: Would you mind reverting and force pushing a new commit that just adds your line, to preserve blame history?

Comment (biteablez) [reply SIGNEES.md:3]: Force push to your branch, of course :)

**Pull request closed without merging**